lldb/include/lldb/API/SBFileSpec.h

```diff
@@ -53,6 +54,8 @@ class LLDB_API SBFileSpec {
 
   uint32_t GetPath(char *dst_path, size_t dst_len) const;
 
+  bool GetPath(lldb::SBStream &dst_path) const;
```

Comment:
I have mixed feelings about adding this API. Generally, we usually use streams when we have potentially multiline output which isn't the case. Here I'm also worried about setting a precedent that will lead to having more methods overloaded with streams. If this were private API I wouldn't mind, but since we guarantee ABI stability for the SB API, once an API has been added, we have to support it forever. 

How about adding a helper to LLDBUtils that takes a `SBFileSpec` and returns the path as a std::string?

Reply:
makes sense, I was trying to avoid the potential problem of the path being longer than `MAX_PATH` but it's still possible to get the real size using `GetFilename` and `GetDirectory`

Reply:
moved to LLDBUtils